spec/mailers/generic_mailer_spec.rb

```diff
@@ -6,18 +6,31 @@
   describe '#drop_duties' do
     let(:duties) do
       user = create(:user)
-      @duties = create_list(:duty, 5, user: user)
+      @duties = [create(:time_range,
```

Comment:
`Time.zone.at(str.in_time_zone.to_i) == str.in_time_zone` lol

Also, I think you can be DRY-er this way:
```ruby
(8..12).map { |n| format('%2d:00", n).in_time_zone }.map { |start_time| create(:time_range, start_time: start_time, end_time: start_time + 1.hour) }
```